freedomofpress/securedrop
# replace Google Authenticator with FreeOTP

## Status

Ready for review

## Description of Changes

Fixes #2705

Google Authenticator is not Free Software, prefer FreeOTP which is available for Android and iOS and tested to work with SecureDrop. It also simplifies the instructions because the interface is the same on both iOS and Android.

## Testing

* Verify the FreeOTP setup instructions at securedrop/journalist_templates/account_new_two_factor.html and docs/servers_2fa.rst work
* Check the URLs at docs/terminology.rst are working
* Verify the images/manual/screenshots/journalist-admin_new_user_two_factor_totp.png screenshot looks good with the updated strings

## Deployment

N/A 

Although there are changes to the deployment instructions, they only relate to the App generating the two-factor authentication token. They can only change deployment if FreeOTP fails where Google Authenticator succeeds.

## Checklist

### If you made changes to the app code:


### If you made changes to documentation:
